packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -212,7 +212,7 @@
 		},
 		"19": {
 			"label": "Forced Switch On Brightness",
-			"description": "Scale is relative to the range set by minimum and maximum brightness level configuration",
+			"description": "Scale is the range defined by minimum and maximum brightness level configuration",
```

I like a mix of the two variants - what do you think?
```suggestion
			"description": "Relative to the range defined by minimum and maximum brightness level configuration",
```
(same for the others if we want to do that).

---

You commented on an old revision. The current wording is

`Value relative to the range defined by minimum and maximum brightness level configuration`

Your version only removes the "Value" part. I guess that is redundant though. I can remove that.